src/components/QuizTemplate.tsx

```diff
@@ -78,7 +81,21 @@ const QuizTemplate: React.FC<QuizProps> = QuizProps => {
   const startQuiz = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
     setIsReady(true);
     const userAnswer = parseInt(e.currentTarget.value);
-    setQuiz(shuffle(filteredQuestions).slice(0, userAnswer));
+    const shuffledQuiz = shuffle(filteredQuestions).slice(0, userAnswer)
+
+    // Shuffle the answer options
+    const choicesArr: string[][] = shuffledQuiz.map(obj => {
```

Comment:
let's go ahead and rename this just for better clarity in the code

```suggestion
    const questionsAndAnswersArr: string[][] = shuffledQuiz.map(obj => {
```

Reply:
I was actually hoping that we could use the one in the global context instead. The naming on this repo seems to be getting out of hand. 😅

Reply:
Okay, thanks for everything.